### PR TITLE
Wishlist Page

### PR DESCRIPTION
**Pull request by AbhishekkGautam**
### Wishlist Page
- Product card
  - 'MOVE TO BAG' button
   - 'close' icon to remove the product from the wishlist.
  
### Screenshot
wishlist

### CODE REVIEW

File: css/navbar.css [hunk 1]
```diff
@@ -84,7 +84,7 @@
   .nav-form .search-btn {
     display: flex;
     align-items: center;
-    padding: 0 15px;
+    padding: 0px 15px;
```
Comment (srishti-maurya) [inline css/navbar.css]: you may write it as `padding: 0 15px;`

**Review by srishti-maurya — APPROVED** [review]
looks good

**Review by manojkumar8400 — APPROVED** [review]
looks good to  me.

**Review by Varu98 — APPROVED** [review]

File: css/navbar.css (hunk 1 above)
Comment (AbhishekkGautam) [reply css/navbar.css]: Done. Thank you :)

**Pull request merged**